src/masternode-payments.h

```diff
@@ -6,6 +6,7 @@
 #ifndef MASTERNODE_PAYMENTS_H
 #define MASTERNODE_PAYMENTS_H
 
+#include "util.h"
```

Comment:
Making util.h the first include ensures that the DBG macro can be enabled on a per file basis.